addons/google_calendar/models/res_users.py

```diff
@@ -96,13 +96,14 @@ def _sync_google_calendar(self, calendar_service: GoogleCalendarService):
         synced_events = self.env['calendar.event']._sync_google2odoo(events - recurrences, default_reminders=default_reminders)
 
         # Odoo -> Google
+        send_updates = not full_sync
         recurrences = self.env['calendar.recurrence']._get_records_to_sync(full_sync=full_sync)
         recurrences -= synced_recurrences
-        recurrences._sync_odoo2google(calendar_service)
+        recurrences.with_context(send_updates=send_updates)._sync_odoo2google(calendar_service)
         synced_events |= recurrences.calendar_event_ids - recurrences._get_outliers()
         synced_events |= synced_recurrences.calendar_event_ids - synced_recurrences._get_outliers()
         events = self.env['calendar.event']._get_records_to_sync(full_sync=full_sync)
-        (events - synced_events)._sync_odoo2google(calendar_service)
+        (events - synced_events).with_context(send_updates=send_updates)._sync_odoo2google(calendar_service, full_sync=full_sync)
```

Comment:
why do you pass `full_sync` to `_sync_odoo2google` here, as `_sync_odoo2google` does not use this parameter ?

Reply:
Thanks for the review.

It had remained from the previous implementation.
Now it's correct.